### PR TITLE
consecutive layout change with 1 button

### PR DESCRIPTION
**Pull request by pymhd**
Check it pls

### CODE REVIEW

File: internal/modules/batt.go [hunk 1]
```diff
@@ -19,11 +19,8 @@ func batt(mo *ModuleOutput, cfg ModuleConfig) {
 	duration, _ := time.ParseDuration(capacity)
 	hours, minutes := fmtDuration(duration)
 	mo.Color = getColor(percentage, cfg)
-	if cfg.ShortFormat {
-		mo.FullText = fmt.Sprintf("%s", status)
-	} else {
-		mo.FullText += fmt.Sprintf("%s %.0f%%/%dh%dm", status, percentage, hours, minutes)
-	}
+	mo.FullText += fmt.Sprintf("%s %.0f%%/%dh%dm", status, percentage, hours, minutes)
+	mo.ShortText = fmt.Sprintf("%s", status)
```
Comment (pymhd) [inline internal/modules/batt.go:23]: No if in every module

File: internal/modules/cpu.go [hunk 2]
```diff
@@ -48,11 +48,7 @@ func cpu(mo *ModuleOutput, cfg ModuleConfig) {
 	cache.Add(cpuTimeKey, newCpuTime, "1h")
 	//Generate output
 	mo.Color = getColor(cpu, cfg)
-	if cfg.ShortFormat {
-		mo.FullText = fmt.Sprintf("%s", mo.FullText)
-	} else {
-		mo.FullText = fmt.Sprintf("%s%.2f%%", mo.FullText, cpu)
-	}
+	mo.FullText = fmt.Sprintf("%s%.2f%%", mo.FullText, cpu)
```
Comment (pymhd) [inline internal/modules/cpu.go:51]: if you want short form as only icon with color no need to change module code at all

File: internal/modules/module.go [hunk 3]
```diff
@@ -24,21 +25,20 @@ func (m *Module) Run(f func(*ModuleOutput, ModuleConfig)) {
...
-				m.Cfg.ShortFormat = false
-			}
-			if m.mute == -1 {
-				m.muteOutput(mo)
-			} else {
+			switch (m.layout +3 ) % 3 {
+			case 0:
+				f(mo, m.Cfg)
+			case 1:
 				f(mo, m.Cfg)
```
Comment (pymhd) [inline internal/modules/module.go:38]: code become simpler

**Review by ilyanix — APPROVED** [review]
Great idea. I like it.

**Pull request merged**